pages/android-walkthrough.md

```diff
@@ -0,0 +1,94 @@
+#Walkthrough: Add a Tangram map to an Android application
+
+With the Tangram Android SDK, you can quickly add beautiful and useful 2D and 3D maps to an Android application. Tangram uses customizable scenes that enable fine control over almost every aspect of your map's appearance, including symbols, lighting, geometry, and feature labels. The Tangram Android SDK is [open source](https://github.com/tangrams/tangram-es), and supports several vector data formats.
+
+Follow this guide to add a Tangram map to your Android application. If you're looking for a simple way to add a pre-configured map and related geographic services to your application, you should check out the [Mapzen Android SDK](https://mapzen.com/documentation/android/).
+
+The Tangram Android SDK can be used with any Android build system that supports Gradle. For this guide we will use [Android Studio](http://developer.android.com/sdk/index.html).
+
+> This guide will assume some knowledge of Android software development. If you want an introduction to writing Android applications, visit the [Android developer site](http://developer.android.com/training/index.html).
```

Comment:
This all looks good! My only very tiny comment: thus far we've mostly used the `>` formatting in the docs to indicate a quote, which is pretty rare – its use here doesn't really cause any problems though, and is kind of a nice "aside" format. (I think normally I just use parentheses.)